Autopilot/Path Manager/MPL3115A2.c

```diff
@@ -29,15 +32,33 @@ char initAltimeter() {
     return altimeterConnected;
 }
 
-void calibrateAltimeter(float altitude){
-    altimeterOffset = altitude;
+void calibrateAltimeter(long int pressure){
+    int msb = pressure >> 9;
```

Comment:
That bit shift stuff is just what I read from the datasheet. Both altitude and pressure are 20 bit, but the way of reading those data are different.
<img width="831" alt="screen shot 2017-08-12 at 7 55 05 pm" src="https://user-images.githubusercontent.com/29065495/29244990-5348992e-7f98-11e7-96fd-bcf510773ac6.png">